### ISSUE TITLE
1.49

### ISSUE THREAD

**Issue by LeiTing-AAA** [dariowouters/ts-fmod-plugin#17]

**Comment by kiry80** [comment]
Update 149 please

**Comment by QQawsl** [comment]
Yes, the game has been updated to the official version of 1.49. The plugin version 1.48.5 is no longer compatible.

**Comment by Piotras42** [comment]
dario wasnt active for 2 months.Maybe its the end of plugin support

**Comment by Piotras42** [comment]
i liked play this game with this sound plugin but without it i will stop playing.Default engine sound are bad.

**Comment by mitchwelters** [comment]
> i liked play this game with this sound plugin but without it i will stop playing.Default engine sound are bad.

fr bro..

**Comment by Piotras42** [comment]
the only optio is roll back ets ver. 1.48.5 and play SinglePlayer.The plugin will work.

**Comment by mitchwelters** [comment]
> the only option is roll back 2 ets ver. 1.48.5 and play SinglePlayer.The plugin will work.

ive ur playing single player you can use a sound mod i used it for tmp

**Comment by Piotras42** [comment]
i play tmp but at the moment until plugin update i have 2 play singleplayer and wersion 1,48.5
